<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>new_sensor</class>
 <widget class="QWidget" name="new_sensor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>198</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Sensor</string>
  </property>
  <widget class="QPushButton" name="load_json_bttn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Load JSON</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save_bttn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sensor_name">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>43</y>
     <width>200</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="adcMin_val">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>73</y>
     <width>70</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="adcMax_val">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>103</y>
     <width>70</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="range_val">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>133</y>
     <width>70</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="flip_travel_check">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>137</y>
     <width>87</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Flip Travel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>45</y>
     <width>100</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sensor Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>75</y>
     <width>100</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ADC Min Value</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>105</y>
     <width>100</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ADC Max Value</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>135</y>
     <width>100</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Range in mm</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
